src/commands/push.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  loadAPICredentials,
```

Comment:
Nitpick: Should this be collapsed into one line, since there is only one import happening?

Reply:
It could be. I figured to keep it multi-line to stay with the style of the other imports. I can change to one-line.